server/server.go

```diff
@@ -471,6 +472,12 @@ func NewServer(userConfig UserConfig, config Config) (*Server, error) {
 			Credentials:    githubCredentials,
 			GithubHostname: userConfig.GithubHostname,
 		}
+
+		githubAppTokenRotator = vcs.NewGithubAppTokenRotator(logger, githubCredentials, userConfig.GithubHostname)
+		err := githubAppTokenRotator.Start()
+		if err != nil {
+			return nil, errors.Wrap(err, "Could not write credentials")
+		}
```

Comment:
atlantis already have scheduled job interface which can handle signal and so on.

Is it possible to integrate this into executor_service?

https://github.com/runatlantis/atlantis/blob/a234bca8287d2c438e7613305171fd9343fb7d75/server/scheduled/executor_service.go#L70

Reply:
Good point, i've added a way to add more jobs to the executor service on the fly during initialisation